components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImpl.java

```diff
@@ -2366,6 +2398,14 @@ private void deleteTransports(Connection connection, String apiID) throws SQLExc
         }
     }
 
+    private void deleteThreatProtectionPolicies(Connection connection, String apiId) throws SQLException {
```

Comment:
Method comment is missing. Shall we add it here? There seems to be multiple places in PR